### ISSUE TITLE
'dot-notation allowKeywords: false' problem

### ISSUE THREAD

**Issue by julianomqs** [jamespamplin/eslint-config-idiomatic#6]
Hello,

I saw you are using the rule `dot-notation` like this:

``` json
"dot-notation": [2, { "allowKeywords": false }]
```

This causes an error (`error  .delete is a syntax error  dot-notation`) in this code:

``` javascript
var express = require( 'express' ),
  router = express.Router();

router.delete( '/:id', controller.destroy );
```

I did Ctrl + F in [IdiomaticJS spec](https://github.com/rwaldron/idiomatic.js/) with `keyword` search term, and it returned no results.
Any special reason to use `allowKeywords`?
Can you reconsider its use? Of course I can override this rule, but maybe others have this same problem, so that's why I created this issue.

Thanks.

**Comment by jamespamplin** [comment]
Sorry for the very late response on this @julianomqs, `allowKeywords: false` was in place to help browser environments, where many of which cannot parse `foo.delete( ... )`.

I've just published `v2.1.0` which uses `eslint:recommended` rules instead of my previous "recommended" set. This ruleset shouldn't include the `dot-notation` rule. You can always use `idiomatic/core` to not use any recommendations though.

**Comment by julianomqs** [comment]
Thanks.
